.github/workflows/main.yml

```diff
@@ -1,23 +1,15 @@
-  
 name: CI
 
-on: 
-  push:
-    branches:
-      - develop
-      - master
-  pull_request:
-    branches:
-      - master
-      - develop
+on: [push, pull_request]
```

Comment:
Without the branch filter we would have ci runs on branches without pull request which are not master or develop

Reply:
Hm, what would be the problem with that? If another branch is chosen for any kind of development/testing, CI results are always helpful IMHO.

Reply:
Well you would have to look quite hard in comparison to find CI results for the branch. Also, if you would have to pay for that one would reduce the amount of ci runs being started.

Reply:
Fair point. There are worker limits for the CI although I never have exceeded them so far. Resolved via 907ea55. :-)